### PR TITLE
[Element 2.1] Show screenshot on terminal (iTerm)

### PR DESCRIPTION
**Pull request by phileba**
resolve https://tricentisflood.atlassian.net/browse/FLO-2027

### CODE REVIEW

File: packages/cli/src/cmd/run.ts [hunk 1]
```diff
@@ -230,6 +230,10 @@ const cmd: CommandModule = {
 				describe: 'Export a HTML report after the test finished running',
 				type: 'boolean',
 			})
+			.option('--show-screenshot', {
+				describe: 'show screenshot in the terminal (iTerm only)',
+				type: 'boolean',
+			})
```
Comment (honglax) [inline packages/cli/src/cmd/run.ts:236]: I think we should add one more property name `default` and set it `false` like
```
.option('--show-screenshot', {
    describe: 'show screenshot in the terminal (iTerm only)',
    type: 'boolean',
    default: false,
})
```

Comment (phileba) [reply packages/cli/src/cmd/run.ts:236]: thank you for your suggestion, in this case, we only care if the `showScreenshot = true`, ignore the rest so the default value is `false` or `undefined` is the same.

Comment (honglax) [reply packages/cli/src/cmd/run.ts:236]: Got it.

**Comment by honglax** [comment]
Hi a @phileba, please resolve all conflicts before merging.

**Review by honglax — APPROVED** [review]

**Pull request merged**